### ISSUE TITLE
Setting xmlns causes problem on children

### ISSUE THREAD

**Issue by zwigglers** [oozcitak/xmlbuilder2#20]
Run `node.end();`
Output
```
<root>
  <child>child text</child>
</root>
```

Run  `node.root().att('xmlns', 'anything').end();`
Output (incorrect)
```
<root xmlns="anything">
  <child xmlns=""/>
</root>
```
Expected output 
```
<root xmlns="anything">
  <child>child text</child>
</root>
```

As you can see xmlns on the child node got set to a blank string, and the text body of child was removed. Setting attributes other than xmlns works properly.

**Version:**
 - xmlbuilder2 [2.1.2]

**Comment by oozcitak** [comment]
I just added a test case to replicate this, but I get the expected output. Can you check please?

**Comment by zwigglers** [comment]
```
const {create} = require('xmlbuilder2');

const node = create({
  root: {
    child: 'child text',
  },
});

const node2 = create().ele('root').ele('child').txt('child text').up().up();

console.log(node.end());
console.log('wrong output', node.root().att('xmlns', 'anything').end());

console.log(node2.end());
console.log('wrong output', node2.root().att('xmlns', 'anything').end());
```

OUTPUT
```
<?xml version="1.0"?><root><child>child text</child></root>
wrong output <?xml version="1.0"?><root xmlns="anything"><child xmlns=""/></root>
<?xml version="1.0"?><root><child>child text</child></root>
wrong output <?xml version="1.0"?><root xmlns="anything"><child xmlns=""/></root>
```
 "dependencies": {
    "xmlbuilder2": "^2.1.2"
  }

**Comment by oozcitak** [comment]
> I'm not sure if there are fixes that have been committed since the 2.1.2

This appears to be the case. It works as intended in the current master branch. I'll release a new version shortly.

**Comment by oozcitak** [comment]
Tested to be fixed in `v2.1.3`.
